cmd/backup.go

```diff
@@ -33,13 +33,15 @@ func init() {
 	backupCmd.Flags().StringVarP(&s3BucketName, "s3-bucket-name", "b", "", "Name of the S3 bucket where to put the actions. Environment variable: DYN_S3_BUCKET_NAME")
 	backupCmd.Flags().StringVarP(&s3BucketFolderName, "s3-bucket-folder-name", "f", "", "Path inside the S3 bucket where to put actions. Environment variable: DYN_S3_BUCKET_FOLDER_NAME")
 	backupCmd.Flags().BoolVarP(&s3DateSuffix, "s3-bucket-folder-name-suffix", "p", false, "Adds an autogenerated suffix folder named using the UTC date in the format YYYY-mm-dd-HH24-MI-SS to the provided S3 folder. Environment variable: DYN_S3_BUCKET_NAME_SUFFIX")
+	backupCmd.Flags().StringVarP(&origin, "origin", "o", "", "The AWS region and profile of the origin target, in the form of profile@region")
+	backupCmd.Flags().StringVarP(&destination, "destination", "d", "", "The AWS region and profile of the origin destination target in the form of profile@region")
```

Comment:
better to not tie the profile and region together; we can have `s3-bucket-region`, `dynamo-table-region`